lib/core/Canvas.js

```diff
@@ -571,7 +651,7 @@ Canvas.prototype.addRootElement = function(rootElement) {
 
   var layer = this.getLayer(layerName, PLANE_LAYER_INDEX);
 
-  svgAttr(layer, 'display', 'none');
+  this.hideLayer(layerName);
```

Comment:
Idea: Accept `layer` as an argument, too.

Reply:
With the API we offer for layers, we never actually return the layer object. We always return the svg group. Using names keeps it consistent with the existing API